actions/store_file_info_MOD.js

```diff
@@ -70,7 +65,8 @@ Variable name:<br>
 
   async action(cache) {
     const data = cache.actions[cache.index];
-    const storage = parseInt(data.storage, 10);
+    const storage = this.eval(data.storage, 10);
+    console.log(storage);
```

Comment:
🤔

Reply:
not sure when that happened hmmmm

Reply:
as for the `The changes on the Store File Info are breaking, not ideal` not sure how to do that without making it a breaking change as storage is always a number... I guess we could just not do the check but that seems wrong.